consul_config.py.ctmpl

```diff
@@ -11,6 +11,7 @@ SECRET_KEY = '''{{template "KEY" "secret_key"}}'''
 SQLALCHEMY_DATABASE_URI = "postgresql://messybrainz@{{.Address}}:{{.Port}}/messybrainz"
 TEST_SQLALCHEMY_DATABASE_URI = "postgresql://msb_test@{{.Address}}:{{.Port}}/msb_test"
 POSTGRES_ADMIN_URI="postgresql://postgres@{{.Address}}:{{.Port}}/template1"
+MB_DATABASE_URI = "postgresql://musicbrainz_ro@{{.Address}}:{{.Port}}/musicbrainz_db"
```

Comment:
This makes messybrainz connect to the master postgres server. Please open a ticket saying that MessyBrainz should use pgbouncer-slave and assign to me.